zoompage.php

```diff
@@ -96,6 +96,9 @@ function zoomPage($id, $urlhash)
 	if ($hideinfo == false || $correcthash == true || User::getCurrent()->isAdmin() || User::getCurrent()->isCheckuser()) {
 		$smarty->assign("showinfo", true);
 	}
+
+	$emailDomain = explode("@", $request->getEmail())[1];
```

Comment:
Yeah, but it's the edge cases that introduce security holes. This I think is fine as it is, but the usage needs the |escape.